tuplex/core/include/logical/ParallelizeOperator.h

```diff
@@ -45,6 +49,12 @@ namespace tuplex {
          */
         std::vector<tuplex::Partition*> getPartitions();
 
+        void setPythonObjects(const std::vector<Partition *>& pythonObjects) { _pythonObjects = pythonObjects; }
```

Comment:
we might need to decide for one getter/setter syntax at some point... good for now.